package.json

```diff
@@ -148,6 +148,7 @@
     "react-router-dom": "^6.24.1",
     "sharedb": "^5.0.2",
     "sharedb-client-browser": "^5.0.2",
+    "styled-components": "^6.1.12",
```

Comment:
Let's not adopt `styled-components` please. We are already all set up with Sass.